### DISCUSSION TITLE
Weekly Lvl60 Raid

### DISCUSSION THREAD

**Discussion by Erson33** [chromiecraft/chromiecraft#5598] [Suggestions]
Hello,

It occurred to me that in the interest of keeping people playing (and testing the content) that a weekly quest for one of the level 60 raids (at random) each week would be pretty effective.

Rewards could be Tier 3 transmog tokens or other interesting cosmetics.

**Comment by balleny** [comment]
while being quite custom, i like the idea. this could offer an easy way for some beginners to join and snag some free loot, a little bit of regression testing, and overall activity

**Reply by Never6** [reply]
Yes but only if the raids are tuned to 70
